src/plugins/validation/2and3/semantic-validators/refs.js

```diff
@@ -15,12 +15,8 @@ module.exports.validate = function({ jsSpec, specStr, isOAS3 }) {
     return messages;
   }
 
-  const basePath = isOAS3 ? ['components', 'schemas'] : ['definitions'];
-
   // Assertation 1
-  // This is a "creative" way to approach the problem of collecting used $refs,
-  // but other solutions required walking the jsSpec recursively to detect $refs,
-  // which can be quite slow.
```

Comment:
Also remove this comment? Also, I don't think the line `// Assertation 1` was supposed to be right above this - will you remove it?